R/fit.R

```diff
@@ -312,10 +318,22 @@ mmrm <- function(formula,
       ))
     }
   }
-
-  covbeta_fun <- h_covbeta_fun(fit)
-  fit$jac_list <- h_jac_list(covbeta_fun, fit$theta_est)
-
+  fit$method <- method
+  if (method == "Satterthwaite") {
+    covbeta_fun <- h_covbeta_fun(fit)
+    fit$jac_list <- h_jac_list(covbeta_fun, fit$theta_est)
+  } else {
+    kr_comp <- h_get_kr_comp(fit$tmb_data, fit$theta_est)
+    fit$kr_comp <- kr_comp
+    if (method == "Kenward-Roger-Linear") {
+      linear <- TRUE
+    } else {
+      linear <- FALSE
+    }
+    w <- solve(fit$tmb_obj$he(fit$theta_est))
+    v_adj <- h_var_adj(fit$beta_vcov, w, kr_comp$P, kr_comp$Q, kr_comp$R, linear = linear)
```

Comment:
for readability better name the arguments of `h_var_adj()` here since there are quite a few